sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -291,6 +292,39 @@ public CosmosClientBuilder keyCredential(CosmosKeyCredential cosmosKeyCredential
         return this;
     }
 
+    /**
+     * Gets the boolean which indicates whether to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If true, this removes the resource from response. It reduces networking
+     * and CPU load by not sending the resource back over the network and serializing it
+     * on the client.
+     *
+     * By-default, this is true.
+     *
+     * @return a boolean indicating whether resource will be included in the response or not
+     */
+    boolean isNoContentResponseOnWrite() {
+        return noContentResponseOnWrite;
+    }
+
+    /**
+     * Sets the boolean to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If set to true, this removes the resource from response. It reduces networking
+     * and CPU load by not sending the resource back over the network and serializing it on the client.
+     *
+     * By-default, this is true.
+     *
+     * @param noContentResponseOnWrite a boolean indicating whether resource will be included in the response or not
+     * @return current cosmosClientBuilder
+     */
+    public CosmosClientBuilder noContentResponseOnWrite(boolean noContentResponseOnWrite) {
```

Comment:
similarly
For other similar configs we are using the Enabled suffix

```java
SessionCapturingOverrideEnabled
ConnectionReuseAcrossClientsEnabled
```
should we change this to `NoContentResponseOnWriteEnabled` to be consistent with the other APIs?

Reply:
This will differ from what we have on .Net side. 
@kirankumarkolli  - what do you think ?